include/swift/Remote/MetadataReader.h

```diff
@@ -29,6 +29,42 @@
 namespace swift {
 namespace remote {
 
+template <typename BuiltType> class FunctionParam {
+  BuiltType Type;
+  StringRef Label;
+
+  union {
```

Comment:
This shouldn't be a union.

Reply:
Ah, sorry, I wanted to change that to be to enum like other flags we have...